### ISSUE TITLE
question about computing loss

### ISSUE THREAD

**Issue by zhangpur** [vvanirudh/social-lstm-pytorch#9]
Thank you a lot for the code. 
I saw the loss function are the sum of negative log-likelihood from Tobs+1 to Tpred  in original paper (equation (4)). But the code in train.py(179) seems to calculate from 2 to Tpred+1 as the loss. I cannot clearly get this, can you give me some help?

**Comment by zhangpur** [comment]
I think I just had some misunderstandings about the code
